docs/development/docker.md

```
@@ -0,0 +1,882 @@
+   ```javascript
+   // webpack.config.js
+   module.exports = {
+     watchOptions: {
+       poll: 1000,
+       ignored: /node_modules/,
+     },
+   };
+   ```
+
+3. **Optimize Watch Patterns**
+   ```yaml
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)